packages/dds/matrix/src/matrix.ts

```diff
@@ -59,6 +63,57 @@ interface ISetOpMetadata {
 	localSeq: number;
 	rowsRefSeq: number;
 	colsRefSeq: number;
+	referenceSeqNumber: number;
```

Comment:
should referenceSeqNumber be optional to signify that op is set by a client who does not know about new format?

Is it better to have ISetOpOld (without both properties) and then

ISetOp = ISetOpOld | (ISetOpOld & {referenceSeqNumber : number, clientId: string;});

(I'm sure there are better way to express it, but the key is that compiler would know that either both properties are present, or both are not, assuming that's what actually we expect)

Reply:
It is local op metadata and is never sent in an op. So, it is local to this client and new clients will have this new interface, while old ones will have old interface, right?

Reply:
Also removed the clientId and just storing ref seq number as client ID is not required.